master/master.cfg

```diff
@@ -113,50 +112,10 @@ for llvm_branch in _LLVM_BRANCHES:
 
 # CHANGESOURCES
 
-# the 'change_source' setting tells the buildmaster how it should find out
-# about source code changes.  Here we point to the buildbot clone of halide.
-
-
-token = Path('github_token.txt').read_text().strip()
-
-
-def pr_filter(pr):
-    # Auto test anything in the halide master repo
-    # print("Considering PR: ", pr['title'], pr['html_url'])
-    # for (k, v) in pr.items():
-    #     print(k, v)
-    repo = pr['head']['repo']
-    result = repo is not None and repo['full_name'] == 'halide/Halide'
-    reviewers = pr['requested_reviewers']
-    if reviewers is not None:
-        for r in reviewers:
-            result = result or (r['login'] == 'halidebuildbots')
-    # Any PR labeled with 'skip_buildbots' should be skipped
-    labels = pr['labels']
-    if labels is not None:
-        for lbl in labels:
-            if lbl['name'] == 'skip_buildbots':
-                result = False
-    print("Testing PR : ", pr['title'], pr['html_url'], ' => ', result)
-    return result
-
+# Here we point the buildbot at third-party codebases, ie. dependencies
+# We use the GitHub mirror of LLVM, pointed at master (main).
```

Comment:
(I thought the GitHub repo is now the truth for LLVM, not just a mirror?)

Reply:
You're right. I'll update the comment.